scripts/protein/esm2/esm2_pretrain.py

```diff
@@ -55,6 +55,7 @@ def main(
     warmup_steps: int,
     limit_val_batches: int,
     val_check_interval: int,
+    log_every_n_steps: Optional[int],
```

Comment:
I see this new arg, which I love the idea of, but I don't see where it's used. I would guess you should pass it into the NeMoLogger thing?